packages/api-derive/src/dappsStaking/index.ts

```diff
@@ -17,23 +15,6 @@ export function stakers(
 ): (contractAddress: ContractAddress) => Observable<AccountId[]> {
```

Comment:
In current main branch this function does not return values. If we remove the whole function any external code using it will throw error. So better to have this stub with depreciation warning rather than removing completely.